src/web/components/Onwards/OnwardsLayout.tsx

```diff
@@ -33,23 +35,27 @@ const decideLayout = (trails: TrailType[]) => {
     }
 };
 
-export const OnwardsLayout = ({ onwardSections }: Props) => (
-    <>
-        {onwardSections.map((onward, index) => (
-            <Flex key={`${onward.heading}-${index}`}>
-                <LeftColumn
-                    showRightBorder={false}
-                    showPartialRightBorder={true}
-                >
-                    <OnwardsTitle title={onward.heading} />
-                </LeftColumn>
-                <OnwardsContainer>
-                    <Hide when="above" breakpoint="leftCol">
+export const OnwardsLayout = ({ onwardSections }: Props) => {
+    const withComments = useComments(onwardSections);
```

Comment:
Just understanding here, is the related content view blocking on fetch for comment counts now?

Reply:
I read you comment, then read my code and for a long time thought that it was blocking, I even refactored the code to fix it. But then I reread the code and realised it actually wasn't blocking, I'd just used variable names that made it read as though it was.

I've fixed the names now.